### ISSUE TITLE
Add meeting agendas to Latest Updates feed

### ISSUE THREAD

**Issue by noahmanger** [18F/fec-cms#933]
So that users can find meeting agendas past and present, add them to the Latest Updates feed. 

---

- By default, agendas should show up in the feed, interspersed with the other update types
- Add a category for "Meeting agendas" in the Publication Type menu under:
![image](https://cloud.githubusercontent.com/assets/1696495/24382302/2b216822-130a-11e7-84f8-50e7c56b0166.png)
- When `update_type=agenda`, only agendas should be shown
- There's no categories, so we don't need to worry about that filter
- Entering a `year` should return agendas for meetings from that year

As for how the update should actually be displayed, that's still being worked out in https://github.com/18F/fec-cms/issues/932 . That will be the last step of this issue once it's finalized.

**Completion criteria:**
- [x] Agendas appear in feed
- [x] Publication type and year filter work
- [x] Show agenda in feed as designed

**Comment by xtine** [comment]
Is there a byline? As in `From: Press Office`? Or should that be blank?

**Comment by noahmanger** [comment]
Sorry, I missed that question earlier. We have a separate issue for designing how they should show up, so it's ok to wait on questions like that for now.

**Comment by noahmanger** [comment]
![design](https://cloud.githubusercontent.com/assets/11636908/24820453/15f13c24-1bb7-11e7-9225-046ac6bd89ad.png) from https://github.com/18F/fec-cms/issues/932

**Comment by noahmanger** [comment]
Resolved by https://github.com/18F/fec-cms/pull/960
